README.md

```diff
@@ -75,64 +81,160 @@ which the standard toast aims to accomplish natively.
+
+```js
+// TODO: write this once we figure out the JS API for supplying HTML messages,
+// and for supplying customizable actions
+```
+
+More complex toasts,
+that don't fit the above content model,
+would require more custom handling on the part of the developer:
+
+```html
+<std-toast>
+  <p>Hello world!</p>
+  <p>A second paragraph?!</p>
+
+  <button>Action 1</button>
+  <button>Action 2</button>
+</std-toast>
+```
 
-### Pseudo classes
+Such an unusual toast would still integrate with other toasts in terms of stacking and positioning behavior,
+and some of the default styles that are inherited may be useful.
+But the page author will need to handle the action clicks themselves (instead of using the `"actionclick"` event),
+and will need to add additional styling to handle the extra contents.
 
-- `:hover`
-- `:focus`
+TODO: when we have a prototype, link to/show an example of this in action.
 
-### Events
+#### Methods
 
-The `<std-switch>` will emit events when it is opened and when it is closed.
-*TODO(me): should it emit an event when the action is taken?*
-*TODO(me): event for onFocus?*
+The `.show(options)` function shows a toast element,
+by toggling its `open=""` attribute to true.
+The `options` include:
+
+- `duration`: how long to show the toast, in milliseconds. Defaults to ???
+- `multiple`: ???
+- `newestOnTop`: ???
+
+TODO: how do `multiple` and `newestOnTop` work?
+Should those be a global setting?
+Per container?
+How do we deal with different toasts having different values?
+
+#### Events
+
+A `<std-toast>` element can fire the following events:
+
+- `"open"`: the toast was shown
+- `"close"`: the toast was closed, either explicitly by the user, or via the timeout.
+  (Note: if animations were applied, the toast may not be entirely invisible at the time this event fires)
+- `"actionclick"`: the toast's call-to-action button or link was clicked, if one exists
+
+### `showToast(message, options)`
+
+The `"std:elements/toast"` module also exports a convenience function,
+`showToast()`,
+which allows creating and showing toasts entirely from JavaScript.
+Behind the scenes,
+`showToast()` creates a `<std-toast>`,
+sets it up using the given message and options,
+inserts it as the last child of `<body>`,
+and then adds the `open=""` attribute,
+to make the toast visible.
+Finally,
+it returns the created `<std-toast>` element,
+allowing further manipulation by script.
+
+`message` is a string that will be inserted as a text node
+(TODO: or as a `<p>` element?) into the created `<std-toast>`.
+
+TODO: can `message` contain HTML?
+
+`options` allows configuring both the attributes of the `<std-toast>`,
+and the options for this particular showing of the toast.
+Thus, the possible options are:
+
+- `theme`, like the attribute
+- `position`, like the attribute
+- `closeButton`, like the attribute
+- `duration`, like the `show()` option
+- `multiple`, like the `show()` option
+- `newestOnTop`, like the `show()` option
```

Comment:
After thinking about this I think you were right, we can probably capture `newestOnTop` in the multiple option, using values like `stackAbove` or `stackBelow`

Reply:
I'll leave this as-is for now since I don't feel sure, but we can work on it as we prototype.